backend/mora/service/orgunit.py

```diff
@@ -593,7 +598,7 @@ def edit_org_unit(unitid):
     to contain the fields that need to change along with the validity dates.
 
     :<json string name: The name of the org unit
-    :<json string parent: The parent org unit
+    :<json string parent: The parent org unit or organisation
```

Comment:
This seems to indicate that we can move units to/from the root level. Should that be the case? Do we test it?

Reply:
Yes, I don't see why not. And no, but I'll add some.

Reply:
My main worry is that whatever we do, we:

1) Know we're doing it.
2) Have tests for it.
3) On some level, it makes sense.

I'm willing to compromise on the last item as long as the two previous are covered 😁  But yes, moving from the root level makes sense, but is somewhat hard to test as it presumes a different hierarchy. Given that the easiest way to test it is moving a unit back and forth, we might as well do both IMO.

Reply:
So you're suggensting we remove the existing validiation rule preventing us from moving from the root level?

Reply:
I think we should ask @alexthirifays — they change seems fairly simple, but I can't quite settle on whether the users would want it. They probably would.

Reply:
And the conclusion from asking him is that it seems a rather risky operation, so out of an abundance of caution we should disallow it, for now. In other words: Re-parenting a unit to or from the root is forbidden.

Reply:
Alright. I'll add an additional validation to ensure we aren't able to move _to_ the root level as well.